app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
Ok! Glad we agree on that - it was a bit heavy weight! I think I will just update docs, and stick to the present situation. I'll open an issue, and we can address this again sometime in the future if required.